### DISCUSSION TITLE
Increase size from 3kb to 4kb in the documentation

### DISCUSSION THREAD

**Discussion by respel** [preactjs/preact#3325] [General]
**Describe the bug**
The website and the docs mention that preact is 3kb in size. However, the latest bundlephobia informs that it's 4kb instead. 

Is there some error there in the calculation of bundlephobia? If not, can we update the docs please?

**Comment by gu-stav** [comment]
It has been mentioned before that 3kB will stay, because that's what preach is aiming for. Apparently v11 will be much closer (maybe even smaller) than 3kB from what I have read.

Also it heavily depends on which features you are using in the end, because unused ones will be tree-shaked away.

**Reply by scarf005** [reply]
> It has been mentioned before that 3kB will stay, because that's what preach is aiming for.

This might confuse users. Consider an extreme case where package "foo" which advertises itself as "1kb" but in fact "5kb"; when asked why they are adverting as "1kb", they reply "because that's what our package is aiming for"; users would be surprised.

> Apparently v11 will be much closer (maybe even smaller) than 3kB from what I have read.

Maybe v11 could do deliver 3 kB. However, as of 2025, 3.5y has passed and v11 hasn't been released yet.

> Also it heavily depends on which features you are using in the end, because unused ones will be tree-shaked away.

`h` and `render` seems to be the minimal required exports to consider an 'app', and they weigh 4.33 kB gzipped, 4.21 kB with brotli according to [bundlejs](https://bundlejs.com/?q=preact&treeshake=[{+h%2Crender+}]).

---

therefore, i still believe preact should advertise itself as `4 kB`, not `3 kB`.
